PolyEngine/Core/Core.vcxproj.filters

```diff
@@ -55,6 +55,9 @@
     <ClCompile Include="Src\BaseObject.cpp">
       <Filter>Source Files\Memory</Filter>
     </ClCompile>
+    <ClCompile Include="Src\Path.cpp">
```

Comment:
If you could additionally add the files to `Core/CMakeFiles.txt` that would be great. Otherwise the build will break again.